kzz7yh-f70880/cod-ve09v2_kzz7yh-f70880.xml

```diff
@@ -457,7 +460,7 @@
                <p xml:id="kzz7yh-f70880-d1e838">
                   <lb ed="#V" n="5"/>Ad primum dicendum quod intellectus possibilis 
       <lb ed="#V" n="6"/>maxime reductus in actum per 
-      <lb ed="#V" n="7"/>similitudinem: non tantummodo est passiuus: sed etiam 
+      <lb ed="#V" n="7"/>similitudinem: non tantum modo est passiuus: sed etiam 
```

Comment:
```suggestion
      <lb ed="#V" n="7"/>similitudinem: non tantummodo est passiuus: sed etiam 
```

Reply:
These can be separate words, but often they get combined when they're next to each other.